src/Imu.cc

```diff
@@ -96,96 +97,98 @@ Errors Imu::Load(ElementPtr _sdf)
   // Load the linear acceleration noise values.
   if (_sdf->HasElement("linear_acceleration"))
   {
-    sdf::ElementPtr elem = _sdf->GetElement("linear_acceleration");
+    sdf::ElementPtr elem = _sdf->GetElement("linear_acceleration", errors);
     if (elem->HasElement("x"))
     {
-      if (elem->GetElement("x")->HasElement("noise"))
+      if (elem->GetElement("x", errors)->HasElement("noise"))
       {
         this->dataPtr->linearAccelXNoise.Load(
-            elem->GetElement("x")->GetElement("noise"));
+            elem->GetElement("x", errors)->GetElement("noise", errors));
```

Comment:
We'll probably want to grab the errors from `Noise::Load` as well, perhaps something like,

```
sdf::Errors noiseErrors = this->dataPtr->linearAccelXNoise.Load(
```

before adding `noiseErrors` into `errors` as well. This probably applies to all the other `Noise::Load` calls below too

Reply:
Good catch! `Load` errors should be grabbed wherever the call happens.